lib/shared/config-manager/src/index.ts

```diff
@@ -27,17 +34,20 @@ export class EnvironmentConfigManager {
     fetchConfigPromise: Promise<void>
     private intervalTimeout?: any
     private disablePolling = false
-    private readonly setConfigBuffer: SetConfigBuffer
+    private clientMode: boolean
+
+    private readonly setConfigBuffer: SetConfigBufferInterface
     private readonly setInterval: SetIntervalInterface
     private readonly clearInterval: ClearIntervalInterface
-    private clientMode: boolean
+    private readonly trackSDKConfigEvent: TrackSDKConfigEventInterface
 
     constructor(
         logger: DVCLogger,
         sdkKey: string,
-        setConfigBuffer: SetConfigBuffer,
+        setConfigBuffer: SetConfigBufferInterface,
         setInterval: SetIntervalInterface,
         clearInterval: ClearIntervalInterface,
+        trackSDKConfigEvent: TrackSDKConfigEventInterface,
```

Comment:
you can combine all three of the lines declaring this field and setting it by just adding `private readonly` to the constructor